pyproject.toml

```diff
@@ -52,7 +52,7 @@ tweepy = "^4.14.0"
 langchain-pinecone = "^0.1.1"
 pinatapy-vourhey = "^0.2.0"
 plotly = "^5.22.0"
-prediction-prophet = { git = "https://github.com/agentcoinorg/predictionprophet.git", rev = "e156a779e648c6775ef301483cef38f1c122813c" }
+prediction-prophet = { git = "https://github.com/agentcoinorg/predictionprophet.git", rev = "63fe84e5c00a992d59f44fe240a3d99debfc703c" }
```

Comment:
`DeployablePredictionProphet*Agent`s now require min 5 websites scraped for their research not to fail